packages-web/calendar/tests/TestProjects/Mendix8/theme/styles/web/sass/core/helpers/_groupbox.scss

```diff
@@ -1,41 +1,16 @@
+//
```

Comment:
Remove all SCS for the theme, they are not needed

Reply:
Done